addons/sys_external/fnc_childrenActions.sqf

```diff
@@ -33,8 +33,8 @@ if (_playerOwnsRadio) then {
         QGVAR(giveRadio),
         _string,
         "",
-        {[(_this select 2) select 0, _target] call FUNC(stopUsingExternalRadio)},
-        {!([(_this select 2) select 0] call FUNC(isExternalRadioUsed))},
+        {[_this select 2, _target] call FUNC(stopUsingExternalRadio)},
+        {!([_this select 2] call FUNC(isExternalRadioUsed))},
         {},
         _params
```

Comment:
Replace with `_radio`.

Reply:
You sure? Isn't this what we mentioned before?

Reply:
You are passing through `[_radio] == _params`, inside ACE3's function that is `_this#2`, so you have to do `_this#2#0`. So you have to pass just `_radio` and you can use `_this#2`.